druid/examples/custom_widget.rs

```diff
@@ -51,54 +55,92 @@ impl Widget<String> for CustomWidget {
         //
         // To check if a dimension is infinite or not (e.g. scrolling):
         // bc.is_width_bounded() / bc.is_height_bounded()
-        bc.max()
+        //
+        // bx.max() returns the maximum size of the widget. Be careful
+        // using this, since always make sure the widget is bounded.
+        // If bx.max() is used in a scrolling widget things will probably
+        // not work correctly.
+        if bc.is_width_bounded() | bc.is_height_bounded() {
+            let size = Size::new(100.0, 100.0);
+            bc.constrain(size)
+        } else {
+            bc.max()
+        }
     }
 
     // The paint method gets called last, after an event flow.
     // It goes event -> update -> layout -> paint, and each method can influence the next.
     // Basically, anything that changes the appearance of a widget causes a paint.
     fn paint(&mut self, ctx: &mut PaintCtx, data: &String, env: &Env) {
-        // Let's draw a picture with Piet!
-
+        println!("{:?}", ctx.size());
```

Comment:
Did you mean to put the println here?

Reply:
no, that was me playing around with the image size.